### PR TITLE
Add script to create Stripe subscription plans

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@alicoding r?

File: ops/stripe_plans.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var async = require("async");
+var currency_data = require("../data/currencies");
+var currencies = Object.keys(currency_data);
```
Comment (ScottDowne) [inline ops/stripe_plans.js]: Maybe make the top two lines one and remove the currency_data var?

Is there something significant to the name "currency_data" or could it be "currenctData"?

Comment (ScottDowne) [inline ops/stripe_plans.js]: I notice you using the underscore in filenames again. I don't really have an issue with it, other than we don't follow that naming convention. If you like the underscore then we can keep it.

**Comment by ScottDowne** [comment]
Looks good to me.

**Pull request merged**